src/Pages/Category/Category.js

```diff
@@ -1,8 +1,120 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Category.scss';
 
 export class Category extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productList: [],
+      isShow: false,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/categoryData.json')
+      .then(result => result.json())
+      .then(categoryData => {
+        this.setState({ productList: categoryData.productList });
+      });
+  }
```

Comment:
```suggestion
  componentDidMount() {
    fetch('/data/categoryData.json')
      .then(result => result.json())
      .then(categoryData => this.setState({ productList: categoryData.productList }));
  }
```
public 경로에서 가져올 경우 url과 포트번호를 생략해주어도 무방합니다